presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/ElasticsearchServer.java

```diff
@@ -35,7 +36,8 @@
     public ElasticsearchServer(String image, Map<String, String> configurationFiles)
             throws IOException
     {
-        container = new ElasticsearchContainer(image);
+        DockerImageName dockerImageName = DockerImageName.parse(image).asCompatibleSubstituteFor("docker.elastic.co/elasticsearch/elasticsearch");
```

Comment:
What’s this?

Reply:
see "Add image compatibility checks" in release notes at https://github.com/testcontainers/testcontainers-java/releases/tag/1.15.0

Reply:
Not sure I understand why we need it here, though. The official images for the open source version of ES are the ones in docker hub.

Reply:
This is called with
- `elasticsearch:7.8.0` from ` TestPasswordAuthentication`
- `elasticsearch:7.0.0` from `TestElasticsearch7IntegrationSmokeTest`
- `docker.elastic.co/elasticsearch/elasticsearch-oss:6.0.0` from `TestElasticsearch6IntegrationSmokeTest`

Reply:
The last one is because there's no 6.0.0 available in docker hub